packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md

```diff
@@ -26,30 +26,321 @@ REST endpoints to configure objects whenever you need. For example:
+
+- A target URI
+- The name of the link relation (`rel`)
+- Other optional properties to help with deprecation, object state or lifecycle management, content negotiation, and so on.
+- Links are implicitly of media type `application/json`. Other media types are only returned in cases where the link isn't an API endpoint.
+
+> **Note:** An object may have multiple links that share the same link relation as shown below for the `"logo"` link.
+
+``` json
+{
+    "_links": {
+        "logo": [
+            {
+              "name": "medium",
+              "href": "https://${yourOktaDomain}/assets/img/logos/groups/active_directory-medium.b3959116154f9d44bd4d0f6b2ae31ea6.png",
+              "type": "image/png"
+            },
+            {
+              "name": "large",
+              "href": "https://${yourOktaDomain}/assets/img/logos/groups/active_directory-large.0e7a58559ac90c4bbc7b33fa14018c50.png",
+              "type": "image/png"
+            }
+         ],
+        "self": { "href": "/example_resource" },
+        "next": { "href": "/page=2" }
+    }
+}
+```
+
+### Links in collections
+
+HAL links returned in a collection of resources may not reflect the total set of operations that are possible on that resource. For example, in a user collection, links indicating that a given user can be unlocked may not be returned and, if returned, may not reflect the correct user state.
+
+Search and list operations are intended to find matching resources and their identifiers. If you intend to search for a resource and then modify its state or make a lifecycle change, the correct pattern is to first retrieve the resource by `id` using the `self` link provided for that resource in the collection. This provides the full set of lifecycle links for that resource based on its most up-to-date state.
+
+### Request debugging
+
+The request ID is always present in every API response and can be used for debugging. You can use this value to correlate events from the [System Log](/docs/reference/api/system-log/) events as well as the [Events API](/docs/reference/api/events/).
+
+The following header is set in each response:
+
+`X-Okta-Request-Id` - The unique identifier for the API request
+
+``` http
+HTTP/1.1 200 OK
+X-Okta-Request-Id: reqVy8wsvmBQN27h4soUE3ZEnA
+```
+
+### Cross-Origin Resource Sharing (CORS)
+
+[Cross-Origin Resource Sharing (CORS)](http://en.wikipedia.org/wiki/Cross-Origin_Resource_Sharing) is a mechanism that allows a web page to make an AJAX call using [XMLHttpRequest (XHR)](http://en.wikipedia.org/wiki/XMLHttpRequest) to a domain that is different from the one from where the script was loaded. Such cross-domain requests would otherwise be forbidden by web browsers, per the [same origin security policy](http://en.wikipedia.org/wiki/Same_origin_policy). CORS defines a [standardized](http://www.w3.org/TR/cors/) way in which the browser and the server can interact to determine whether to allow the cross-origin request.
```

Comment:
```suggestion
[Cross-Origin Resource Sharing (CORS)](http://en.wikipedia.org/wiki/Cross-Origin_Resource_Sharing) is a mechanism that allows a web page to make an AJAX call by using [XMLHttpRequest (XHR)](http://en.wikipedia.org/wiki/XMLHttpRequest) to a domain that is different from the one where the script was loaded. Such cross-domain requests would otherwise be forbidden by web browsers, in accordance with the [same origin security policy](http://en.wikipedia.org/wiki/Same_origin_policy). CORS defines a [standardized](http://www.w3.org/TR/cors/) way in which the browser and the server can interact to determine whether to allow the cross-origin request.
```